### PR TITLE
Pp66 cards home page

### PR DESCRIPTION
**Pull request by ofeterovskaya**
1. Add cards components to the home page
2. Add a new folder cards to the public folder
3. Add imgs into the card folder

### CODE REVIEW

**Comment by ofeterovskaya** [comment]
ops, I just realized I forgot to add  links to the filtered market page. I''ll do it later today

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Oksana! Great job! 
one small issue: according our model 
image
there is category "Sport Card", not "Sport Card**s**" The same issue with the "Yu-Gi-Oh" name. Could you please fix this typos?

File: components/CardsHome.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+"use client";
+import React, { useEffect, useState } from "react";
+import Link from "next/link";
+import { Grid, Card, CardContent, CardMedia, Typography } from "@mui/material";
+
+export default function CardsHome() {
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
```
Comment (AnnaPestova1) [inline components/CardsHome.jsx]: do we need this useEffect here?

Comment (ofeterovskaya) [reply components/CardsHome.jsx]: fixed and did new commit

Comment (ofeterovskaya) [reply components/CardsHome.jsx]: Yu-Gi-Oh! now is on

**Review by AnnaPestova1 — APPROVED** [review]

**Review by AnnaSolovykh — APPROVED** [review]

**Pull request merged**